<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Password_manager</class>
 <widget class="QWidget" name="Password_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password_manager</string>
  </property>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed Light</family>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="login_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed</family>
    </font>
   </property>
   <property name="text">
    <string>Login:</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed</family>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed Light</family>
    </font>
   </property>
   <property name="text">
    <string>Show existing passwords...</string>
   </property>
  </widget>
  <widget class="QLabel" name="tittle_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Semi Condensed Black</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Password Manager v.01</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pass_gen_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed Light</family>
    </font>
   </property>
   <property name="text">
    <string>Generate password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
